components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -79,9 +90,9 @@ export default function Preferences() {
                             placeholder="e.g. https://github.com/username/dotfiles"
                             onChange={(e) => setDotfileRepo(e.target.value)}
                         />
-                        <button className="secondary ml-2" onClick={() => actuallySetDotfileRepo(dotfileRepo)}>
```

Comment:
**thought**: Feels so much better we don't have to use an extra onClick function here. 💭